hack/update-api-reference-docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Strictly speaking, we do not need this file.
I added it to keep the hack/verify-A.sh, hack/update-A.sh convention.